assignment-2/assignment2.rb

```diff
@@ -0,0 +1,220 @@
+=begin
+1. Customize the room_cleaner program.
+Possibilities:
+Accept user input to get room names (or offer a selection of available rooms)
+Create an invoice-style output at the end of the program that varies based on the rooms and processes completed.
+=end
+
+
+
+rooms = []
+puts "Enter a room, i.e. bedroom, kitchen, bathroom, livingroom, "
+select_room = gets.chomp
+
+rooms.push(select_room)
+
+
+def dust(room)
+  puts "The #{room} has been dusted"
+end
+
+def vacuum(room)
+  puts "The #{room} has been vacuumed."
+end
+
+def make_bed
+  puts "The bed has been made."
+end
+
+def mop_floor
+  puts "The floor has been mopped."
+end
+
+def wash_dishes
+  puts "The dishes have been washed."
+end
+
+def clean_toilet
+  puts "The toilet has been cleaned."
+end
+
+def clean_room(room)
+  if room == "bedroom"
+    vacuum(room)
+    make_bed
+  elsif room == "kitchen"
+    mop_floor
+    wash_dishes
+  elsif room == "bathroom"
+    mop_floor
+    clean_toilet
+  else
+    puts "The #{room} requires no specific cleaning processes."
+  end
+
+  mop_floor if room == "kitchen" || room == "bathroom"
```

Comment:
`mop_floor` again?

Reply:
Over ambitious with cleaning, told the robot to tame it down, done.